CODEOWNERS

```diff
@@ -1 +0,0 @@
-* @actions/actions-runtime
```

Comment:
do you want to add yourself or ci-app-libraries as owners here?

Reply:
True, I will add @datadog/ci-app-tracers

Reply:
we'd probably have to add ci-app-tracers to this repo: 
<img width="924" alt="Screenshot 2023-01-04 at 11 23 48" src="https://user-images.githubusercontent.com/22798219/210534753-bc40868e-4a31-4cf5-bfa1-a47a86b5ed2c.png">

Reply:
hanged to @datadog/ci-app-libraries